packages/web/src/utils/escapeArguments.ts

```diff
@@ -0,0 +1,7 @@
+export const escapeArguments = (args: string[]) => {
+  return args.map(arg => {
+    const escapedArg = arg.replace(/(")/g, '\\$1');
```

Comment:
Still quoting of `'` missing (and not important: maybe backtick, new line), i.e. for `args: ["abc'def"]` it will output `abc'def` instead of `"abc'def"` or `abc\'def`

Maybe the simplest fix would be to change regex below to quote if `[\s']` is matching